site/en/docs/web-platform/navigation-api/index.md

```diff
@@ -214,61 +190,64 @@ This key remains the same even if the current entry's URL or state changes.
+
+In the Navigation API, you can call the `.getState()` method of the current entry (or any entry) to return a copy of its state:
+
+```js
+console.log(navigation.current.getState());
+```
 
-In the App History API, you can call the `AppHistoryEntry.getState()` method of the current entry (or any entry) to return a copy of its state.
 By default, this will be `undefined`.
-You can synchronously set the state for the current `AppHistoryEntry` by calling:
+You can synchronously set the state for the current `NavigationEntry` by calling:
 
 ```js
-appHistory.updateCurrent({state: something});
+navigation.updateCurrent({state: something});
 ```
 
-You can also set the state when navigating programmatically with `appHistory.navigate()` (this is [described below](#programmatic-navigation)).
+You can also set the state when navigating programmatically with `navigation.navigate()` (this is [described below](#programmatic-navigation)).
 
-In the App History API, the state returned from `.getState()` is a copy of the previously set state.
+In the Navigation API, the state returned from `.getState()` is a copy of the previously set state.
 If you modify it, the stored version won't also change.
 For example:
 
 ```js
-appHistory.updateCurrent({state: {count: 1}});
+navigation.updateCurrent({state: {count: 1}});
 
-const state = appHistory.current.getState();
+const state = navigation.current.getState();
 state.count = 2;
 
-console.info(appHistory.current.getState()); // count will still be one
+console.info(navigation.current.getState().count); // will still be one
 ```
 
 ### Access All Entries
 
 The "current entry" is not all, though.
-The API also provides a way to access the entire list of entries that a user has navigated through while using your site via its `appHistory.entries()` call, which returns a snapshot array of entries.
+The API also provides a way to access the entire list of entries that a user has navigated through while using your site via its `navigation.entries()` call, which returns a snapshot array of entries.
 This could be used to, e.g., show a different UI based on how the user navigated to a certain page, or just to look back at the previous URLs or their states.
 This is impossible with the current History API.
 
-It's possible to handle events on `AppHistoryEntry`.
-These events include "dispose" (if the entry is no longer accessible), along with "navigateto" and "navigatefrom".
-For example, you might add a "navigatefrom" handler to clean up some state when the user leaves a specific page (either by forward navigation or their Back or Forward button).
+You can also listen for a "dispose" event on individual `NavigationEntry`s, which is fired when the entry is no longer part of browser history. This can happen as part of general cleanup, but also happen when navigating. For example, if you traverse back 10 places, then navigate forwards, those 10 history entries will be disposed.
```

Comment:
```suggestion
You can also listen for a "dispose" event on individual `NavigationHistoryEntry`s, which is fired when the entry is no longer part of browser history. This can happen as part of general cleanup, but also happen when navigating. For example, if you traverse back 10 places, then navigate forwards, those 10 history entries will be disposed.
```

here and throughout

Reply:
Didn't realise this had changed. Will change the other references too